### PR TITLE
Define preventSilentAccess() behavior

### LINKED ISSUE #565: preventSilentAccess() -- what effect does calling it have?
The webauthn spec does not mention `preventSilentAccess()`.  However, the credman spec [does](https://w3c.github.io/webappsec-credential-management/#framework-credential-management).

What does it mean for public key credentials?  If it has a meaning, let's explicitly state that.  If it doesn't, let's make it throw an error.

### PR DESCRIPTION
**Pull request by selfissued**
Fixes #565

***
<a href="https://pr-preview.s3.amazonaws.com/selfissued/webauthn/pull/758.html" title="Last updated on Jan 24, 2018, 7:11 PM GMT (0714e95)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/758/1e77b42...selfissued:0714e95.html" title="Last updated on Jan 24, 2018, 7:11 PM GMT (0714e95)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1371,6 +1371,18 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 </div>
 
+###  Preventing silent access to an existing credential - PublicKeyCredential's `[[ preventSilentAccess]](credential, sameOriginWithAncestors)` method ### {# preventSilentAccessCredential}
```
Comment (emlun) [inline index.bs]: Should there be a space in `[[ preventSilentAccess]]`?

File: index.bs [hunk 2]
```diff
@@ -1371,6 +1371,18 @@ When this method is invoked, the user agent MUST execute the following algorithm
...
+
+<div link-for-hint="PublicKeyCredential/[[ preventSilentAccess]](credential, sameOriginWithAncestors)">
```
Comment (emlun) [inline index.bs]: Ditto

File: index.bs [hunk 3]
```diff
@@ -1371,6 +1371,18 @@ When this method is invoked, the user agent MUST execute the following algorithm
...
+
+Calling the <dfn for="PublicKeyCredential" method>\[[ preventSilentAccess]](credential, sameOriginWithAncestors)</dfn> method
```
Comment (emlun) [inline index.bs]: Should there be a space in `[[ preventSilentAccess]]`? There's also a `\` just before it which shows up in the rendered text.

File: index.bs [hunk 4]
```diff
@@ -1371,6 +1371,18 @@ When this method is invoked, the user agent MUST execute the following algorithm
...
+will have no effect on authenticators that require a user ceremony,
```
Comment (emlun) [inline index.bs]: Perhaps `[=authorization gesture=]` instead of `user ceremony`?

**Comment by emlun** [comment]
Oh! Indeed it is. I was confused because it's not there in references to the methods, only the definitions it seems. Never mind that comment, then.

**Comment by AngeloKai** [comment]
Thanks for writing it up. Everything looks good here, though I'd prefer the language to include something along the line of "It's recommended that when this API is called, the browsers show a console log message."

**Pull request merged**